slapp/transfers/upload.py

```diff
@@ -2,16 +2,10 @@
 import datetime
 import tempfile
 import slapp.transfers.utils as utils
+import slapp.utils.query_utils as query_utils
 
 
 class UploadSchema(argschema.ArgSchema):
-    sqlite_db_file = argschema.fields.InputFile(
-        required=True,
-        description="sqllite input db file")
-    sql_table = argschema.fields.Str(
-        required=False,
-        missing="manifest_table",
-        description="table where the manifests are stored")
     sql_filter = argschema.fields.Str(
```

Comment:
@njmei and I discussed this filter as being too poorly-defined and possible subject to some unwanted postgres operations. We think changing to expect a list of `roi_manifests.id` and a harcoded SQL query `SELECT manifest FROM roi_manifests WHERE id IN {}` is better. This puts the burden on the upload user to figure out which `roi_manifests.id` values to upload.

Reply:
made the change to the code and the PR description comments.